frontend/src/boot/components.ts

```diff
@@ -4,11 +4,13 @@ import BaseButton from 'components/BaseButton.vue';
 import BaseInput from 'components/BaseInput.vue';
 import BaseSelect from 'components/BaseSelect.vue';
 import LoadingSpinner from 'components/LoadingSpinner.vue';
+import ProgressIndicator from 'components/ProgressIndicator.vue';
```

Comment:
(arbitrary location)

> the UI flashes back to showing the loading indicator briefly after scanning is completed, but it's not clear to me why that is.

what's happening here is:
- we set `scanStatus` to complete
- this triggers the `AccountReceiveTable` component to be mounted [here](https://github.com/ScopeLift/umbra-protocol/blob/0a06f0ca72d473c66747a4a7116d76ab23c254f6/frontend/src/pages/AccountReceive.vue#L79), and that component is passed an array of announcements returned from scanning
- the `AccountReceiveTable` component then shows _it's own loading spinner_, because it does async post-processing of the announcements [here](https://github.com/ScopeLift/umbra-protocol/blob/0a06f0ca72d473c66747a4a7116d76ab23c254f6/frontend/src/components/AccountReceiveTable.vue#L409-L429)

Reply:
maybe in `AccountReceiveTable` we change the message from "Scanning for funds..." to "Almost done...." or something a bit different/distinguished from the previous "scanning for funds" message

Reply:
Another option would be to set it to the progress indicator with 100% so it doesn't appear to flash at all. Feels a little hacky but might better UX. What do you think? Gonna try it out and see how it feels.

Reply:
Hmm, there's a good chance in the future that we no longer can reliably show a progress indicator (e.g. if scanning is done in a web worker or wasm), so we'd be removing this whole thing and make sure to remove it there too. I think my preference would be to change the copy to:

- Step 1: Fetching annoucements...
- Step 2: Checking for funds...
- Step 3: Processing result...

And that way it carries over nicely to a future version where we can't show a progress bar in step 2

Reply:
I think a webworker and/or wasm version will still be able to report status updates so we can show completion percentage. That said, after trying it out, I actually don't like my own suggestion. It makes it seem like it's "stuck" at 100%. Went with your suggestion and updated to say "Processing results..."